### DISCUSSION TITLE
Decrypt Password

### DISCUSSION THREAD

**Discussion by rbf121** [GerryDazoo/Slinger#296] [Ideas]
I figured out a way to decrypt the passwords found in the Slingplayer Desktop console.log.  Before I post steps, I'm wondering if someone wouldn't mind being a test subject.  Does anyone have both the 15 character password that works in Slinger and the hashed password found in the log and wouldn't mind sharing to validate this works?  You can just share the hashed pw and then I will post the password to confirm it's a match.  (could also do this over DM)

The hashed password is something like this, followed by 64 characters.
"password":"E1:

Edit: Slinger v3.0.8f can handle the encrypted passwords now.

**Comment by Mikoshi-nyudo** [comment]
password="E1:5733E985252008DC29D20ED21E21E1ED45243B5C027A3F1725131DF2F2D4CA74"

**Reply by rbf121** [reply]
Hm, the formatting looks different so I'm guessing this is not correct but is the password, Edit: removed confirmed pw

There is something called hashKey which I'm unsure if it's required or differs per install.

**Reply by Mikoshi-nyudo** [reply]
Yes! well done. It is from the SP app SBProfile.xml.. same encryption/hashing though.

**Reply by Mikoshi-nyudo** [reply]
So I wonder if it might eventually it be possible from a JTAG NAND dump to then be able to read that Hashed/Encrypted password from the Solo or Pro boxes.... as I would assume they store it on the box in that format too. Would allow users to get into Setup Assistant again, and maybe FW hex hack update like ProHD if that's eventually needed too.

**Reply by sp222222** [reply]
man ! this might be the whole key to unbricking IF the spm1.5 tab with admin password takes this found cracked password you speak of from JTAG NAND

**Reply by bunklung** [reply]
Can someone explain to me the difference between the 64 character hash, the clear text, and the 15 character password Slinger uses?

I know my clear text admin password, but it's not working in Slinger. I did not pull this 15 character password from the Sling service, but I do have backups of xxxxx_Slingboxes.dat on both Android and the WebSlingPlayer. They both show adminPassword: hashes, but I think they are salted so they are different between these two files. Even if you decrypt this, will this mean I still can't use Slinger for lack of a 15 character password?

**Comment by almoney** [comment]
Very interesting.  Now we need to find out how to generate the E1 password and unlock the reset slingboxes.

**Reply by almoney** [reply]
I have an E1: password for a new 350.  could you try to decode?

E1:028500CA99D23BDC00EA62B380DB1643

Thank you!

**Reply by almoney** [reply]
<Device dbid="{4F5B5AA5-243A-4533-BEB7-219FF850BA6B}" username="admin" password="E1:028500CA99D23BDC00EA62B380DB1643" name="My Slingbox" port="5001" address="" useaddress="0" last.address="" last.port="0" id="AD7351388D176D67658B9A740ECF225D" - is the complete info from SBProfile.xml

**Reply by almoney** [reply]
E1:6A8DB04C2049208BE8DB1D632CAF621C4D18537FB3FD03E9DDD7ED881430445B is one of the Sling configured passwords.

**Reply by rbf121** [reply]
Not sure what to do with the shorter one but the 64 character one produced, 9LcmOVX5wjRu0fi

Is that correct?

**Reply by almoney** [reply]
Yes that is correct!  The short one is interesting to me since it is a new box.  Maybe pad it with 00's?

**Reply by almoney** [reply]
Are you willing to share the code?

**Reply by almoney** [reply]
In one of my copys of the SBProfile.xml the password is only E1:4DFCC5727534B14E  so it does change them.

**Reply by GerryDazoo** [reply]
Is it possible you had an old Solo/Pro? That might explain the shorter password. The long E1: is the hash for a 15 character PW.

**Reply by almoney** [reply]
The HD PRO has this password: E1:028500CA99D23BDC795D3F45D52BA42B admin seems to let you configure, but you never complete setup.  (I do get a remote saved, but the box remains in setup mode while working with slinger.)

**Reply by rbf121** [reply]
Both of the shorter 32 character ones come back with "admin"

**Reply by almoney** [reply]
OK

**Reply by rbf121** [reply]
Actually, the really short one, E1:4DFCC5727534B14E was "dmin".  And to your question, there is no code to share since this is more of a process.  But let me dig into it more and see if there is a better way to do it.

**Reply by PeteSki1618** [reply]
I have 3 HDPros but I keep getting a password error when I run VLC.  The console.log file lists one with the password as "admin" and the other two have E1 numbers, neither of which are 64 characters (one 48 and one 32).  I am posting them both here and, if possible, can someone try to decrypt them for me to see if they will work?  I would greatly appreciate it!!!  If not, is there a way to retrieve the passwords from the boxes?  I have not run the Desktop App for years (the last date listed in it was 2018) so the passwords are probably not valid any more anyway, but worth a shot.  Thanks in advance for your help!
Here are the two numbers:  E1:A8307F4C1EF38CC9D924090C57D3DF8AB796BE588DE07ADA  (48 char)
        E1:107802A0D90529889EC1FAF582A235D8   (32 char)

**Reply by rbf121** [reply]
Haven't been following this closely since I don't have one but weren't the Pro HD models bricked if you left them online?  I doubt this is a password issue then.

"password":"mathteach"
"password":"philski"

**Reply by GerryDazoo** [reply]
rbf121: Yes they were bricked if left online unprotected. The difference between it and the Pros and Solos is you can do a factory reset and log in with the default password "admin". We're not so lucky with the other boxes.

**Reply by RandomMan719** [reply]
If anyone knows how to do the JTAG Nand dump, I would try it on my unbricked solos and pro, to see if we could decrypt the password I set in them.  I assume you have to get a JTAG device or something, but I could also just solder the pins from the board to a serial connection, if that is what is needed, I don't know anything about using JTAG. 

Maybe I should start a JTAG Disscussion.

**Reply by GerryDazoo** [reply]
RandomMan: Not sure that would advance the cause. I'd like to get my hands on the firmware that got downloaded to the bricked box. Just to see if there might be a new default admin password.  It must be online somewhere but you'd need the file name. Right now to get that info somebody might have to sacrifice a Solo for the cause and do a packet capture while it's getting bricked. 
Unless someone could whip up a Solo simulator to send the new firmware check request.
Right now I'm not convinced that the brick firmware would even have a new password. If it was me, I would have disabled all logins.

**Reply by PeteSki1618** [reply]
First, thank you rbf121 for decrypting them for me.  Those were the passwords that I knew and tried.  Evidently, I am one of the guys that left my boxes on and now I have 3 Pros that are dead.  It just kept giving me a Password error so I thought that's what was wrong.  If anyone figures out a way to "unbrick" them, PLEASE let me know!  Thanks to all for your help.  
I just borrowed an M1 from a friend and got that to stream.  Working on getting the Remote to work so I might be back on asking for help again.  In the meantime, thanks again to all for your help.

**Reply by JoeM2016** [reply]
Check out #265 They have a procedure to unbrick Pro HD's.

**Reply by PeteSki1618** [reply]
Thanks JoeM2016.  I took a quick look but am not too sure what to do besides doing a hard reset.  Will give it another look tomorrow.  Thanks for the info.  I appreciate it!

**Reply by PeteSki1618** [reply]
I have finally been able to revive an M1, modified one of the remotes to work with my cable box, and can successfully view it remotely from outside the house.  Thank you SO much to all for your help.  I still have 3 ProHD's that are bricked and still cannot get to work.  Thanks JoeM2016 for referring me to #265.  I tried it on a Win11 machine with both versions of SlingPlayer with no luck.  I will see if I can set up a virtual Win7 machine and try it there.  At least I have ONE working box (but it's my friend's so may have to give it back which is why I'd like to get the Pros up and running.  Thanks again!

**Comment by sp222222** [comment]
It’s simple : you need a valid administrator password to utilize the
slinGER process/script.

They have cracked the 350/500
password so sit tight until that genius can get things in order for all to
enjoy the spoils.

**Reply by bunklung** [reply]
Hmm, then my brain is not remembering the password correctly, cause it's not working when I use the less than 15 character clear text password I had. Maybe my Slingbox is not working either? Do you have to put this clear text password in quotes?
;sbtype="240"
password=mypassword
or
password="mypassword"
; uncomment the next lines ipaddress and port with your local network info

**Reply by rbf121** [reply]
Try,
dHmOwKpGGaa2CZi

4u8wnxR3kQtaXVd

**Reply by bunklung** [reply]
Okay, I was expecting them to be the same decrypted 15 character string. I figured there was some salting going on. Anyhow, the first one worked! It would be interesting to understand how you decrypt it down to the original clear text password for the users that do NOT have old log/dat files. Then they could encrypt this to the 15 character string needed for Slinger. I found mine since I happen to have a old app backups from a rooted phone. I also had an old PC backup and found the dat file that way.

Thank you so much for your help and effort!

**Comment by RandomMan719** [comment]
rbf121 can you decrypt these for me?

I got two, the first one is a M2

E1:A21B6C268C42FA3773DC92B007C1838CEC3531C09E553D5672A154F5A8682ECA

and Here's a 350

E1:f83dbf17634f9b7e354bca30d8a796309b2e572bcf1dde1084c90638031c8d68

**Reply by rbf121** [reply]
33pl5y1xdf0uelc
6ogxanVwf0B71MQ

**Reply by RandomMan719** [reply]
Correct

**Comment by mdehrlich** [comment]
Am I missing something?  Slinger works with the long encrypted password.  What value is there in the cleartext password?

**Reply by rbf121** [reply]
You can use slinger with the long password (64 characters coming after E1:)?  I think folks were only able to use the 15 character adminPassword with slinger, if they retrieved them before the servers shutdown.  What device are you using which will take the encrypted password?  It did not work on the M1 I tested.

The benefit of this is that after shutdown, you cannot get the adminPassword unless you find it in a log file somewhere.  And some people are reporting only seeing the E1: passwords in the log.  So for those folks, it may be possible to find the encrypted password (64 characters for 350/500/M1, may be shorter for other devices) in localstorage of devices where Slingplayer software was run (SlingPlayer Desktop, etc) and decrypt them to the password used in slinger.  This is more of a recovery benefit for someone who did not get their passwords in time.

**Reply by mdehrlich** [reply]
ok.  Thanks for clarifying this.

**Reply by RandomMan719** [reply]
Does @GerryDazoo know, if the E1: password works with Slinger?

**Reply by GerryDazoo** [reply]
The login message to the slingbox only has room for max 16 characters for the password. So I would think not.

**Comment by Drano32** [comment]
Hi rbf121, I have a 500 - E1:f36eed87be76c343db4ef73b32bd5866f9d39694ab89a1898ddc37f490464985

and an M1 - E1:5687D58440C92654746BA400CAA51A8AACB1A286770B663E82BD6D9A000193E4

I thought I knew the password for both but apparently not!

**Reply by Drano32** [reply]
And a third one, my other 500 - E1:66e9eaac35142e3af74ab5ff7ff748644362c45ca6d5924a00072ad60e534725

**Reply by rbf121** [reply]
What device do you have that will take the 64 character E1: password through slinger?

**Reply by rbf121** [reply]
Someone said it but I was not able to do it myself on the M1.  I suppose it's possible it could work on other devices.

**Reply by RandomMan719** [reply]
> Someone said it but I was not able to do it myself on the M1.  I suppose it's possible it could work on other devices.

Deleted thanks

**Reply by Drano32** [reply]
I've now tried the three long encrypted E1 passwords and they just don't work on my two 500s and M2 - RBF121, if you can decrypt them for me it would be greatly appreciated!  If easier by DM in the Discord server let me know.

**Comment by Drano32** [comment]
Have not tried the 64 character strings on my 500s (and the other box is actually an M2 not an M1). I could try using these strings - but would appreciate the unencrypted versions greatly though!

**Reply by rbf121** [reply]
Here is the first one, having some trouble getting the others right now.  Will try again later.
"password":"o35QjNNpj3PnYsw"

**Reply by Drano32** [reply]
Thank you rbf121, will try this one now!

**Reply by Drano32** [reply]
First one helped me use Slinger successfully for 1st time, it revived one of my 500s!  Thank you so much.  I'll double check the other two to make sure that I copied all of the text.

**Reply by Drano32** [reply]
Remaining ones (confirmed that I copied all of the text):

E1:66e9eaac35142e3af74ab5ff7ff748644362c45ca6d5924a00072ad60e534725

E1:5687D58440C92654746BA400CAA51A8AACB1A286770B663E82BD6D9A000193E4

First one is a 500, second one is an M2.

**Reply by rbf121** [reply]
"password":"iBbjMEe3BnnJXZQ"
"password":"805fjwaw3cy6f8h"

**Reply by Drano32** [reply]
This has made my weekend, thank you again!

**Comment by sp222222** [comment]
Can you change channels on that M1 ? I thought we could not
configure inputs.

**Reply by PeteSki1618** [reply]
Yes.  I have full functionality.  I had to look at the codes the remote was sending and by a lot of trial and error, got the correct codes to send to my Optimum cable box.

**Reply by mobrad0729** [reply]
Rbf121.  Would it be possible to provide me with the short password for these devices?  I guess I was hoping that someone would pick up the slingbox product and didn't start looking at alternatives until too late.  

350
E1:cfa4adb5e5ad43c6bccce47f5a8a485772e0512a345e1b469a1df6811958ed3f

M1
E1:35CC7A5ECD62CC6B83C689E48C415E5F7EF3EF85577BE471D40894462EEA17B7

I also have a third one which I don't have immediate access to but has the following 
E1:98A216F66ABB721EBA605E2F9F519F52CB50C57DE753E930998C516A06BC4C77

Appreciate

**Reply by PeteSki1618** [reply]
I jinxed myself.  Yesterday I had full functionality on the remote both in the house and remotely.  Today, I tried resurrecting my ProHD's so was playing around with them with no luck.  Just hooked the M1 back up and now the remote doesn't work.  Checked all wires.  Any ideas would be greatly appreciated?

**Comment by AME-Miguel** [comment]
Rbf121. Would it be possible to provide me with the short password for these devices please? 

E1:52A8C588D0A7EB32ECD85E491C57FC6DBBCEDC1CC5033D709674363E0690A304
E1:DA7BF99114593B89DC3433615B3CCAE31D0A892F86F5ABAD

Thanks in advance

**Reply by AME-Miguel** [reply]
They are both M1s

**Reply by tom76017** [reply]
In windows, look at:   **_%appdata%\Sling Media\Slingplayer Desktop\Logs\console.log_**
I found my M1 password there.  Search for **_"admin","PASSWORD"_**

**Reply by AME-Miguel** [reply]
Thanks, tom76017 but the passwords I found in this console.log did not work for me. That is why I was hoping sending the following hash passwords for my two M1s would help rbf121 generate the working slinger passwords:

E1:52A8C588D0A7EB32ECD85E491C57FC6DBBCEDC1CC5033D709674363E0690A304
E1:DA7BF99114593B89DC3433615B3CCAE31D0A892F86F5ABAD

Could you please rbf121? Thanks

**Comment by mobrad0729** [comment]
I may not have put this at the right level.  

Rbf121. Would it be possible to provide me with the short password for these devices? I guess I was hoping that someone would pick up the slingbox product and didn't start looking at alternatives until too late.

350
E1:cfa4adb5e5ad43c6bccce47f5a8a485772e0512a345e1b469a1df6811958ed3f

M1
E1:35CC7A5ECD62CC6B83C689E48C415E5F7EF3EF85577BE471D40894462EEA17B7

I also have a third one which I don't have immediate access to but has the following
E1:98A216F66ABB721EBA605E2F9F519F52CB50C57DE753E930998C516A06BC4C77

Appreciate

**Reply by rbf121** [reply]
5LMeofsMykxnZtU
o3oenj9mo3829o3
jf5oozq0776q4vs

**Comment by AME-Miguel** [comment]
rbf121, could you please provide me with the short passwords fort these M1 devices please?

E1:52A8C588D0A7EB32ECD85E491C57FC6DBBCEDC1CC5033D709674363E0690A304
E1:DA7BF99114593B89DC3433615B3CCAE31D0A892F86F5ABAD

thanks

**Reply by rbf121** [reply]
chichariro14
miguelito7

**Comment by tryin100** [comment]
> 
I have four 500 units with these "E1" passwords.
Any help you could provide decrypting them would be greatly appreciated.
Thanks

"password":"E1:e25150b2d57c34ff9a99b90b7cd6154aeb3532ab4d02da6c1d97764c01179170"
"password":"E1:4a0a847d370147999b8ad73680c1ab7359fc25895572ca278a041b9a04e7978f"
"password":"E1:c57927202c6bdeeafded9a771e5d03418ffaacda2bcc88070804ae318e9097ac"
"password":"E1:e90952e28e6570f4cb75f21ed27ecc178d7c50d1900cf3470f05cc7c98a6cad1"

**Reply by rbf121** [reply]
nuaIOi8TukPyEsd
x4d0ReWcfHZkGWX
rjxAeks8EepC7iv
XS2wMPB1lQhlYvf

**Reply by tryin100** [reply]
these worked perfectly.  thank you so much!

**Reply by tryin100** [reply]
I understand that the new slinger code will decrypt the E1 passwords so they can be used, but could you decrypt this one last E1 password for me so I have a complete set of decrypted passwords for all my 500s?  Thanks
password":"E1:2da72e95d08a877c2a201679dac615cf330642b9c620c9d7ca95105976a1287a"

**Comment by rbf121** [comment]
It took quite a while to figure out but I'm finally able to decrypt/encrypt passwords using python code.  It leverages the Slingplayer DLL code to call the DecryptPassword method directly.  I'll clean it up and post the code when I have a chance.  There may be some other functions/methods in there of use.

**Reply by rbf121** [reply]
This one seems interesting, for the bricked boxes:

	def FirmwareUpload(self, fileName=defaultNamedNotOptArg, bRestartAfterUpdate=defaultNamedNotOptArg):
		'Firmware bin file will be uploaded to the box'
		return self._oleobj_.InvokeTypes(14, LCID, 1, (24, 0), ((8, 1), (3, 1)),fileName
			, bRestartAfterUpdate)

**Reply by RandomMan719** [reply]
@rbf121  Nice Work!

**Reply by GerryDazoo** [reply]
rbf121: Can you email me directly about how your using the DLL. We might be able unbrick the ProHDs much easier with this.
gerry@dazoo.net.

**Comment by rein-10** [comment]
so i have an m1 and following the e1 its only 48 characters. You think you can help me?  E1: 8870BAD2EE0965778870BAD2EE0965778EFE3D3FC7CA57D3

**Reply by rbf121** [reply]
Encrypted Password: [E1:8870BAD2EE0965778870BAD2EE0965778EFE3D3FC7CA57D3]
Decrypted Password: [holahola10]

**Comment by rbf121** [comment]
I posted the python code to run the DecryptPassword method via WebSlingPlayer DLL/COM methods here:
https://github.com/rbf121/Slinger-Utility

There may be some other useful methods in there for other utility purposes, if someone can figure out how to use them.
Specifically, these seem interesting,
```
	def FirmwareUpload(self, fileName=defaultNamedNotOptArg, bRestartAfterUpdate=defaultNamedNotOptArg):
		'Firmware bin file will be uploaded to the box'
```
and
```
	def UploadIRBinFile(self, fileName=defaultNamedNotOptArg):
		'IR bin file will be uploaded into box, current input must not be tuner.'
```

I also had a brainstorming session with Gerry and it should be possible to update Slinger app to support the E1: encrypted passwords in the config.ini as well.  I believe he will be rolling that out in a future release which will eliminate the need for this utility method.

**Comment by rick624** [comment]
Hi rbf121, I have an E1 encryption password from my Pro-HD, could you provide me decrypt the short password please?
Thanks
E1:AAC746203E72E6BF8478491D63966225

**Reply by GerryDazoo** [reply]
E1:AAC746203E72E6BF8478491D63966225-> mnc453

**Reply by rick624** [reply]
Thanks GerryDazoo, This decrypted password is really my old admin password,
But my Pro-HD is already bricked Orz

**Reply by GerryDazoo** [reply]
If it's bricked you need to do a factory reset. New password will be "admin"

**Reply by rein-10** [reply]
mine is also my old admin password but it wont work. Any advice?

**Comment by rein-10** [comment]
never mind just saw another post that i can reset it and use admin. I dont have a problem with it not having an ir blaster since i have a harmony in that room

**Reply by PeteSki1618** [reply]
I have 3 bricked ProHD's.  I have tried doing a factory reset but then Slingplayer doesn't find it.  I've read through several of these threads to try and figure out how to revive it.  Does anyone have a step-by-step installation procedure for dummy's just like the PDF someone posted for setting up the server?  I got my friend's M1 up and running with full functionality but not the ProHD's.  I tried on a Win11, Win10 and XP computer.  Someone mentioned needing a Win7 PC which I don't have.  Any help would be greatly appreciated.  Thanks in advance.  Happy Thanksgiving to all!

**Reply by Mikoshi-nyudo** [reply]
> I have 3 bricked ProHD's. I have tried doing a factory reset but then Slingplayer doesn't find it. I've read through several of these threads to try and figure out how to revive it. Does anyone have a step-by-step installation procedure for dummy's just like the PDF someone posted for setting up the server? 

https://github.com/GerryDazoo/Slinger/discussions/357

**Reply by PeteSki1618** [reply]
Thanks.  I'll give it a try.

**Comment by Toei79** [comment]
question i did a reset on one 350, then i not longer can use remote? guess Gerry already answer me this but try see this progressing here.

my issue i only can hear audio in my 350 but not video from composite maybe its component so i only get audio but idk if its

and where i get encrypted password? i check rbf121 slinger utility documents anyway

edit i get those , i have three boxes. two 350 and one sb300
any help with those too? thank you so much!

E1944C1591041CD07947996757282636F2E78D3643F34652
E1:886bc39716e311ecc47a8ed97d9907d57adbee2661f41134ae4a9f0846e91aa7
E1:ef8e2c1dea5275a7d1563da8d4e57824ed464caffb1feb34255300a241840c88

i tried those in the .ini file but not working.

**Reply by GerryDazoo** [reply]
You're probably using the wrong VideoSource

**Comment by GerryDazoo** [comment]
The latest version of slingbox_server support using the E1: password. The slinger code will do the decryption.

**Reply by Toei79** [reply]
> The latest version of slingbox_server support using the E1: password. The slinger code will do the decryption.

how you add it? i mean, you need start E1: And large numbers letters, or just large numbers letters without E1:
i tried without success unfortunately

**Reply by GerryDazoo** [reply]
I thought the release notes and my announcement here about the new version made it very clear you need the E1:
i.e.  password=E1:DA7BF99114593B89DC3433615B3CCAE31D0A892F86F5ABAD

**Reply by Toei79** [reply]
OK yea, i just add Password

[SLINGBOX]
sbtype="350/500/M1/M2"
;sbtype="Solo/Pro/ProHD"
;sbtype="240"
password=E1:ef8e2c1dea5275a7d1563da8d4e57824ed464caffb1feb34255300a241840c88
 

here but its not working.  the finderid match with this password its a 350 but keep get this error. 

![11-28-2022 (222322)](https://user-images.githubusercontent.com/79808610/204431111-c0246dd9-0afe-4a63-bca3-6ed0bd3df8ac.jpg)

and i have one that i reset using adm password its working. still just audio in that one.

**Reply by GerryDazoo** [reply]
Toei: The finderid is NOT the password, sorry,

**Reply by Toei79** [reply]
well but im using the password not the finderid. 

i have three slingboxes,  when i run .py it gives me finderid of the one im running, then i compare with the one from console log, so i know which one im using the one that match finderid.  i can do a video showing the .ini file and command running.

for what im understand you see finderid, password on the console.log. they are different of course,

**Comment by rein-10** [comment]
Found an app that lets you see the logs of an android smartphone. Viewing the log I found something that looks as a key. The problem is that when ever I use the new server i get an error of missing characters and when trying utility by itself i get some strange characters. This is what i think is my key but im not sure. de4d7fa50352a19abb99d48eaa5873511eee11afb342f9d37c55a1c3b1008b53

And next is the full line of code

#EXT-X-KEY METHOD=AES-128,URI="sling://de4d7fa50352a19abb99d48eaa5873511eee11afb342f9d37c55a1c3b1008b53"

**Comment by tryin100** [comment]
I understand that the new slinger code will decrypt the E1 passwords so they can be used, but could you decrypt this one last E1 password for me so I have a complete set of decrypted passwords for all my 500s? Thanks
password":"E1:2da72e95d08a877c2a201679dac615cf330642b9c620c9d7ca95105976a1287a"

**Reply by lepmail** [reply]
Try: **C3VyCcaM0wzbnQY**

**Reply by tryin100** [reply]
thank you

**Comment by Toei79** [comment]
Can you decryp mines too?, I'm unable to make work so far honestly 

E1:E1944C1591041CD07947996757282636F2E78D3643F34652
E1:886bc39716e311ecc47a8ed97d9907d57adbee2661f41134ae4a9f0846e91aa7
E1:ef8e2c1dea5275a7d1563da8d4e57824ed464caffb1feb34255300a241840c88

Thank you

**Reply by socelillyhammer** [reply]
@Toei79 if you understand and know how to read and run the coding the slingbox_server.py has the coding to decryt the PY (from what I can read and semi-understand).  Search "decryt" to find the script in the py file

So if you are more coder tech savy than me you could , i assume, extract that coding, and run it somewhere to decrypt and you will see your pw.

I dont know how to do this stuff but if I could I would help.

**Reply by Toei79** [reply]
I just paste the password  to the  .ini file the one large starts with E:

For what you said i need check "decryt" in the .ini file

I thought  it was just paste in password  in the .ini file , 

I wish know more , sometimes  I give up and back to see it I fix things there.


Here its where I place it

![Screenshot_20221129-174234_Samsung Internet](https://user-images.githubusercontent.com/79808610/204664624-3366fb7b-55de-4293-a1b6-d89297c2853d.jpg)

**Reply by GerryDazoo** [reply]
Yes, that's the right place put in the E1: password there
E1:blahblahblah
Make sure you leave in the E1:

You do not have to hack the code as soccilliy suggests.

**Reply by salahkanaan** [reply]
Hi. Can you decrypt mine.
E1:646608a8c13707ae850fecfe20f34d75b97555fcbd3dc3a0a32f14c43b4a05
E1:563deba5114b2ee5ea0a41aa76e4ea288879c650bfb9c5faa793f859786d2c0
I have the 350 model.
thanks a lot

**Comment by salahkanaan** [comment]
> I figured out a way to decrypt the passwords found in the Slingplayer Desktop console.log. Before I post steps, I'm wondering if someone wouldn't mind being a test subject. Does anyone have both the 15 character password that works in Slinger and the hashed password found in the log and wouldn't mind sharing to validate this works? You can just share the hashed pw and then I will post the password to confirm it's a match. (could also do this over DM)
> 
> The hashed password is something like this, followed by 64 characters. "password":"E1:
> 
> Edit: Slinger v3.0.8f can handle the encrypted passwords now.

Hi. Can you decrypt mine.
E1:646608a8c13707ae850fecfe20f34d75b97555fcbd3dc3a0a32f14c43b4a05
E1:563deba5114b2ee5ea0a41aa76e4ea288879c650bfb9c5faa793f859786d2c0
I have the 350 model.
thanks a lot

**Reply by salahkanaan** [reply]
Also this one please
E1:08a8c137050fecfe20£34d75b9645 d3dc3aOaea32f364a64

**Comment by GerryDazoo** [comment]
You don't need to decrypt it. The server will do it for you. Just set it as your password.

**Comment by salahkanaan** [comment]
Many thanks. I did reset the box. VLC is opening up with no errors. But screen is black and not streaming.

**Reply by GerryDazoo** [reply]
without you posting the server output it's hard to say what's happening. Make sure you have the correct videosource selected.

**Comment by salahkanaan** [comment]
Many thanks. All works fine. In my case, No need to decrypt the password which I do not have. I have reset my slingbox 350 to default and put the below configuration with "admin" as password and works like a charm. Many thanks for this script.
[SLINGBOX]
sbtype="350/500/M1/M2"
;sbtype="Solo/Pro/ProHD"
;sbtype="240"
password=admin
And it would better to input the IP and port addresses
